unified_range/api.py

```diff
@@ -16,13 +16,13 @@ def from_semver(semver_spec: str) -> str:
     return str(ver_rng)
 
 
-def to_semver(spec: str) -> str:
+def to_semver(spec: str, separator: str=' ') -> str:
```

Comment:
I would recommend to have the new comma separated feature as a new function on the api and not as an optional arg on the `to_semver` function.

Reply:
added `to_semver_comma_separated`